dipy/denoise/localpca_slow.py

```diff
@@ -0,0 +1,118 @@
+import numpy as np
+import scipy as sp
+
+def localpca_slow(arr, sigma, patch_radius=1, rician=True):
+    r"""Local PCA Based Denoising of Diffusion Datasets
+
+    Parameters
+    ----------
+    arr : A 4D array which is to be denoised
+    sigma : float or 3D array
+        standard deviation of the noise estimated from the data
+    patch_radius : The radius of the local patch to
+                be taken around each voxel
+    rician : boolean
+        If True the noise is estimated as Rician, otherwise Gaussian noise
+        is assumed.
+
+    Returns
+    -------
+
+    denoised_arr : 4D array
+        this is the denoised array of the same size as that of
+        arr (input data)
+
+    References
+    ----------
+    .. [Manjon13] Diffusion Weighted Image Denoising Using Overcomplete Local PCA
+                  Manjon JV, Coupe P, Concha L, Buades A, Collins DL
+
+    """
+
+    if arr.ndim == 4:
+
+        tou = 2.3 * 2.3 * sigma
```

Comment:
This should be the variance, not the standard deviation, so your docstring is misnommed or you missed the square. See see http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0073021#s2

~~~
After an exhaustive search for the optimum value of τ parameter, it was set to (2.3σ)2, where σ2 is the estimated local noise variance
~~~